README.md

```diff
@@ -1,2 +1,49 @@
 # observability-data-model
 Documentation for telemetry data model and semantic conventions
+
+## Logging
+
+Current logging data model:
+- [ViaQ Data Model](https://viaq.github.io/documentation/data_model/public/data_model.html)
+
+Relevant OTEL models:
+- [Logs Data Model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md)
+- [Kubernetes Resource Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md)
+
+Current OTEL support is Tech Preview, JSON over HTTP only.
```

Comment:
Could we add some pointers to docs how to enable this? Which version supports it?

Reply:
Will do.